product.json

```diff
@@ -27,7 +27,8 @@
 		"reportIssueUrl": "https://github.com/Microsoft/sqlopsstudio/issues/new?labels=customer%20reported%20issue",
 		"requestFeatureUrl": "https://github.com/Microsoft/sqlopsstudio/issues/new?labels=feature-request"
 	},
-	"releaseNotesUrl": "https://go.microsoft.com/fwlink/?linkid=862039",
+	"gettingStartedUrl": "https://go.microsoft.com/fwlink/?linkid=862039",
+	"releaseNotesUrl": "https://go.microsoft.com/fwlink/?linkid=875578 ",
```

Comment:
there's a trailing space in the url here, not sure if it matters

Reply:
thanks, that's a typo.  Didn't seem to matter, but I'll correct.